free_fleet_adapter/free_fleet_adapter/nav1_robot_adapter.py

```diff
@@ -49,11 +49,14 @@
 
 class Nav1TfHandler:
 
-    def __init__(self, robot_name, zenoh_session, tf_buffer, node):
+    def __init__(self, robot_name, zenoh_session, tf_buffer, node,
+                 robot_frame='base_footprint', map_frame='map'):
         self.robot_name = robot_name
         self.zenoh_session = zenoh_session
         self.node = node
         self.tf_buffer = tf_buffer
+        self.frame_robot = robot_frame
+        self.frame_map = map_frame
```

Comment:
```suggestion
        self.robot_frame = robot_frame
        self.map_frame = map_frame
```
nit: it is more easily understood as a noun, instead of misunderstanding it as `<verb>_<noun>`